Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -1144,8 +1144,10 @@
         return;
 
     RefPtr<Event> coreEvent;
-    if (m_lastMouseEvent)
-        coreEvent = MouseEvent::create(eventNames().clickEvent, &frame->windowProxy(), platform(*m_lastMouseEvent), { }, { }, 0, 0);
+    if (event || m_lastMouseEvent) {
+        auto platformEvent = event ? WTFMove(*event) : platform(*m_lastMouseEvent);
+        coreEvent = MouseEvent::create(eventNames().clickEvent, &frame->windowProxy(), platformEvent, { }, { }, 0, 0);
```

Comment:
(I know this isn't new to this patch but it just reminded me, this should probably be changed to be a PointerEvent at some point)